packages/lib/storybook/stories/internals/Button.stories.tsx

```diff
@@ -55,36 +62,44 @@ export const Default: StoryObj = {
     }
 };
 
-export const CopyIconOnlyButton: StoryObj = {
+export const CopyIconOnlyButton: StoryObj<CopyIconButtonProps> = {
     render: args => {
         return (
-            <CoreProvider
-                loadingContext={'test'}
-                i18n={
-                    new Language({
-                        locale: 'en-US',
-                        translations: {
-                            'button.copy': 'Copy',
-                            'button.copied': 'Copied!'
-                        }
-                    })
-                }
-                resources={global.resources}
-            >
+            <CoreProvider {...coreProps}>
                 <CopyIconButton {...args} text={'Text to be copied'} />
             </CoreProvider>
         );
     },
     parameters: {
-        controls: { exclude: ['useSessions', 'countryCode', 'shopperLocale', 'amount', 'showPayButton'] }
+        controls: { include: ['disabled', 'inline', 'ariaLabel'] }
+    },
+    args: {
+        disabled: false
+    }
+};
+
+export const PaymentButton: StoryObj<PayButtonProps> = {
+    render: args => {
+        return (
+            <CoreProvider {...coreProps}>
+                <PayButton {...args} onClick={() => console.log('Pay button clicked')} />
+            </CoreProvider>
+        );
+    },
+    parameters: {
+        controls: { exclude: ['useSessions', 'countryCode', 'shopperLocale', 'showPayButton'] }
     },
     argTypes: {
-        disabled: { control: 'boolean' },
-        inline: { control: 'boolean' },
-        ariaLabel: { control: 'text' }
+        amount: { control: 'object' },
+        secondaryAmount: { control: 'object' }
     },
     args: {
-        disabled: false
+        amount: { value: 1000, currency: 'EUR' },
+        secondaryAmount: { value: 1200, currency: 'USD' },
```

Comment:
Did you verify if we are handling it correctly for the Bulgarian symbol? Is it normal to show `BGN` (as mentioned in the ticket) ?